.circleci/config.yml

```diff
@@ -111,6 +111,31 @@ jobs:
       - run: composer functional
       - run: composer integration
 
+  # PHP 7.2 test with HIGHEST dependencies:
+  #   Determines whether a newer version of a dependency has broken Drush.
+  test_72_highest:
+    <<: *defaults
+    docker:
+      - image: wodby/php:7.2
+        environment:
+          - MYSQL_HOST=127.0.0.1
+          - UNISH_DB_URL=mysql://root:@127.0.0.1
+      - image: circleci/mysql:5.7.24
+    steps:
+      - checkout
+      - run: cp .docker/zz-php.ini /usr/local/etc/php/conf.d/
+      - run: composer remove --dev webflo/drupal-core-strict --no-update
```

Comment:
We can convert to drupal/core-recommended and drupal/dev-dependencies now, but let's do that as follow-on work.

Reply:
Once we convert, we won't have to just remove here.